packages/react/src/manifest.json

```diff
@@ -1,16 +1,16 @@
 {
-  "name": "Example App",
+  "name": "React JS Scaffold",
   "author": {
-    "name": "John Doe",
-    "email": "support@zendesk.com",
+    "name": "Daniel Montes",
+    "email": "daniela.montes@zendesk.com",
     "url": "https://www.zendesk.com"
   },
   "defaultLocale": "en",
   "private": true,
   "location": {
     "support": {
       "ticket_sidebar": {
-        "url": "assets/iframe.html",
+        "url": "http://localhost:3000/",
```

Comment:
I haven't tested this, but it looks like this wouldn't work in production - how are we handling this?

Reply:
So the main purpose for this is to have a better developing experience so the app has two stages one for development and the other for production, this value will be change automatically once we build the production version. (This is mention in the README, one of the big changes here from the previous version is that in this one we have two stages one for development and one for production, hope this still match the goal of the react scaffold )

Reply:
Got it. I _think_ that's fine. I'll have a play around with it to see if it causes any unexpected issues.